src/index.html

```diff
@@ -4,9 +4,491 @@
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews">
+            Reviews: 5
+          </p>
+        </div>
+
+        <div class="card__price_box" >
+          <p class="card__price">
+            Price:
+          </p>
+
+          <p class="card__price_value" >
+            $2,199
+          </p>
+        </div>
+        
+        <a 
+          href="#" 
+          class="card__button" 
+          data-qa="hover"
+        >
+          buy
+        </a>
+      </div>
+
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img 
+          class="card__img"
+          src="./images/imac.jpeg"
+          alt="iMac"
+        >
+
+        <div class="card__stars_box" >
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">
+            Product code: 195434
+          </p>
+
+          <div class="stars--4 stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews">
+            Reviews: 5
+          </p>
+        </div>
+
+        <div class="card__price_box" >
+          <p class="card__price">
+            Price:
+          </p>
+
+          <p class="card__price_value" >
+            $2,199
+          </p>
+        </div>
+        
+        <a 
+          href="#" 
+          class="card__button" 
+          data-qa="hover"
+        >
+          buy
+        </a>
+      </div>
+
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img 
+          class="card__img"
+          src="./images/imac.jpeg"
+          alt="iMac"
+        >
+
+        <div class="card__stars_box" >
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">
+            Product code: 195434
+          </p>
+
+          <div class="stars--4 stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews">
+            Reviews: 5
+          </p>
+        </div>
+
+        <div class="card__price_box" >
+          <p class="card__price">
+            Price:
+          </p>
+
+          <p class="card__price_value" >
+            $2,199
+          </p>
+        </div>
+        
+        <a 
+          href="#" 
+          class="card__button" 
+          data-qa="hover"
```

Comment:
you need set data-qa="nav-hover" (not just hover) to the 4th nav link for testing (Laptops & computers), data-qa="card" to the first card and data-qa="card-hover" (not just hover) to the link Buy inside the first card, the rest are superfluous

Reply:
thanks, i already fixed it